src/client/components/mainPageHeader/logo.svg

```diff
@@ -0,0 +1,26 @@
+  <path stroke="#33363a" stroke-miterlimit="10" stroke-linejoin="round" stroke-linecap="round" stroke-width="3" fill="none" id="svg_19" d="m12.543477,69.704137c-1.151,0 -2.14,0.418 -2.968,1.256c-0.825,0.836 -1.24,1.859 -1.24,3.064l0,16.939c0,1.559 0.561,2.734 1.674,3.533c1.113,0.797 2.304,1.082 3.571,0.848c1.189,-0.232 2.023,-0.662 2.506,-1.285c0.479,-0.623 0.72,-1.654 0.72,-3.096l0,-16.939c0,-1.205 -0.414,-2.229 -1.239,-3.064c-0.824,-0.838 -1.832,-1.256 -3.024,-1.256z" class="st26"/>
+  <path stroke-miterlimit="10" stroke-linejoin="round" stroke-linecap="round" stroke-width="3" stroke="#33363a" fill="none" id="svg_20" d="m25.734634,54.903782c-1.152,0 -2.142,0.42 -2.965,1.258c-0.825,0.834 -1.239,1.84 -1.239,3.008l0,39.484c0,1.129 0.393,1.977 1.181,2.539c0.784,0.566 1.797,0.85 3.023,0.85c1.271,0 2.295,-0.283 3.079,-0.85c0.791,-0.562 1.184,-1.41 1.184,-2.539l0,-39.485c0,-1.168 -0.414,-2.174 -1.239,-3.008c-0.825,-0.837 -1.834,-1.257 -3.024,-1.257z" class="st26"/>
+  <path stroke-miterlimit="10" stroke-linejoin="round" stroke-linecap="round" stroke-width="3" stroke="#33363a" fill="none" id="svg_21" d="m38.647522,59.659136c-1.191,0 -2.207,0.428 -3.052,1.283c-0.844,0.854 -1.268,1.889 -1.268,3.096l0,36.037l-0.001,0c0,1.133 0.403,1.979 1.211,2.541c0.804,0.566 1.842,0.848 3.109,0.848c1.228,0 2.236,-0.281 3.024,-0.848c0.788,-0.562 1.181,-1.408 1.181,-2.541l0,-36.037c0,-1.207 -0.412,-2.242 -1.24,-3.096c-0.824,-0.855 -1.814,-1.283 -2.964,-1.283z" class="st26"/>
+  <path stroke="#33363a" stroke-miterlimit="10" stroke-linejoin="round" stroke-linecap="round" stroke-width="3" fill="none" id="svg_22" d="m53.829096,37.639c-1.368318,-0.311 -2.715054,0.059 -4.039129,1.111c-1.326233,1.049 -1.990968,2.531 -1.990968,4.439l0,58.408l0,0c0,2.143 1.532344,3.213 4.601348,3.213c3.065767,0 4.599189,-1.07 4.599189,-3.213l0,-58.408c-0.001079,-3.271 -1.057533,-5.121 -3.170441,-5.55z" class="st26"/>
+  <path stroke-miterlimit="10" stroke-linejoin="round" stroke-linecap="round" stroke-width="3" stroke="#33363a" fill="none" id="svg_24" d="m34.553,101.422l0,0c-0.001,0 0,0.014 0,0.021l0,-0.021z" class="st26"/>
+  <path stroke-miterlimit="10" stroke-linejoin="round" stroke-linecap="round" stroke-width="3" stroke="#33363a" fill="none" id="svg_25" d="m106.007,89.312c0,3.314 2.689,6.002 6.006,6.002c4.144,0 7.505,-3.359 7.505,-7.504c0,-5.182 -4.201,-9.381 -9.383,-9.381c-6.477,0 -11.725,5.248 -11.725,11.727c0,8.094 6.562,14.656 14.657,14.656" class="st26"/>
+ </g>
+</svg>
```

Comment:
missed empty line at the end of the file.

Reply:
done)